doc/articles/uno-development/api-extensions.md

```diff
@@ -0,0 +1,36 @@
+# API Extensions
```

Comment:
I'm particularly interested in comments on this document

Reply:
Seems great - will we able to use this for eg the map control?

Reply:
We could. There's a need to make sure that interfaces are available for the override, but it's the overall idea.